my-app/src/components/login/login.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import '../../app.css';
+import { Link, withRouter } from 'react-router-dom';
+
+
+function validate(username, password) {
+  const errors = [];
+
+  if (username.length < 6) {
+    errors.push('Username should be at least 5 characters long');
```

Comment:
If the username.length is equal to 5, isn't the Username at least 5 characters long?

Reply:
I didn't write this, it just shows up as my change because I was fixing ESLint errors for jsx in non jsx files